.github/config/xb1.json

```diff
@@ -3,6 +3,9 @@
   "__comment" : "TODO: Deploy a runner and change this",
   "docker_runner_service": "runner-xb1",
   "runner_tag": "xb1",
+  "on_device_test": {
```

Comment:
looks like `main_win.yaml` is missing the block of code that starts with `on-device-test:` that is present in `main.yaml`

It's a bit less than ideal that we let those drift so much and we should try and consolidate both into a single flow, but for now i'd just copypaste that section